Makefile

```diff
@@ -463,23 +494,49 @@ operator-sdk: $(LOCALBIN)
+	sed -i 's@containerImage: .*@containerImage: $(IMG)@' config/manifests/$(BUNDLE_VARIANT)/bases/opentelemetry-operator.clusterserviceversion.yaml
+	sed -i 's/minKubeVersion: .*/minKubeVersion: $(MIN_KUBERNETES_VERSION)/' config/manifests/$(BUNDLE_VARIANT)/bases/opentelemetry-operator.clusterserviceversion.yaml
+
+	$(OPERATOR_SDK) generate kustomize manifests -q --input-dir $(MANIFESTS_DIR) --output-dir $(MANIFESTS_DIR)
+	cd $(BUNDLE_DIR) && cp ../../PROJECT . && $(KUSTOMIZE) build ../../$(MANIFESTS_DIR) | $(OPERATOR_SDK) generate bundle $(BUNDLE_BUILD_GEN_FLAGS) && rm PROJECT
+
+	# Workaround for https://github.com/operator-framework/operator-sdk/issues/4992
+	echo "" >> bundle/$(BUNDLE_VARIANT)/bundle.Dockerfile
+	echo "LABEL com.redhat.openshift.versions=v$(MIN_OPENSHIFT_VERSION)" >> bundle/$(BUNDLE_VARIANT)/bundle.Dockerfile
+	echo "" >> bundle/$(BUNDLE_VARIANT)/metadata/annotations.yaml
+	echo "  com.redhat.openshift.versions: v$(MIN_OPENSHIFT_VERSION)" >> bundle/$(BUNDLE_VARIANT)/metadata/annotations.yaml
+
+	$(OPERATOR_SDK) bundle validate $(BUNDLE_DIR)
 	./hack/ignore-createdAt-bundle.sh
-	./hack/add-openshift-annotations.sh
-	$(OPERATOR_SDK) bundle validate ./bundle
+
+.PHONY: bundle
+bundle:
+	BUNDLE_VARIANT=community VERSION=$(VERSION) $(MAKE) generate-bundle
+	BUNDLE_VARIANT=openshift VERSION=$(VERSION) $(MAKE) generate-bundle
+
 
 .PHONY: reset
 reset: kustomize operator-sdk manifests
 	$(MAKE) VERSION=${OPERATOR_VERSION} set-image-controller
-	$(OPERATOR_SDK)  generate kustomize manifests -q
-	$(KUSTOMIZE) build config/manifests | $(OPERATOR_SDK) generate bundle -q --overwrite --version ${OPERATOR_VERSION} $(BUNDLE_METADATA_OPTS)
-	$(OPERATOR_SDK) bundle validate ./bundle
-	./hack/ignore-createdAt-bundle.sh
-	./hack/add-openshift-annotations.sh
+	$(OPERATOR_SDK)  generate kustomize manifests -q --input-dir config/manifests/community --output-dir config/manifests/community
+	$(OPERATOR_SDK)  generate kustomize manifests -q --input-dir config/manifests/openshift --output-dir config/manifests/openshift
+
+	$(KUSTOMIZE) build config/manifests/community | $(OPERATOR_SDK) generate bundle $(BUNDLE_GEN_FLAGS) --kustomize-dir config/manifests/community --output-dir bundle/community
+	$(KUSTOMIZE) build config/manifests/openshift |$(OPERATOR_SDK) generate bundle $(BUNDLE_GEN_FLAGS) --kustomize-dir config/manifests/openshift --output-dir bundle/openshift
+
+	# Workaround for https://github.com/operator-framework/operator-sdk/issues/4992
+	echo "" >> bundle/community/metadata/annotations.yaml
+	echo "  com.redhat.openshift.versions: v$(MIN_OPENSHIFT_VERSION)" >> bundle/community/metadata/annotations.yaml
+	echo "" >> bundle/openshift/metadata/annotations.yaml
+	echo "  com.redhat.openshift.versions: v$(MIN_OPENSHIFT_VERSION)" >> bundle/openshift/metadata/annotations.yaml
+
+	$(OPERATOR_SDK) bundle validate ./bundle/community
+	$(OPERATOR_SDK) bundle validate ./bundle/openshift
+	rm bundle.Dockerfile
```

Comment:
Why removing the bundle dockerfile? It was moved to ./bundle/<variant>

Reply:
It is generated again